doc/whats_new/v1.4.rst

```diff
@@ -43,6 +43,16 @@ Changelog
     :pr:`123456` by :user:`Joe Bloggs <joeongithub>`.
     where 123456 is the *pull request* number, not the issue number.
 
+
+Code and Documentation Contributors
```

Comment:
I think this is a merge error. Can we remove this diff?

Reply:
Done.